SimpleWAWS/Controllers/TemplatesController.cs

```diff
@@ -39,5 +46,18 @@ public IEnumerable<BaseTemplate> Get()
             list.Add(WebsiteTemplate.EmptySiteTemplate);
             return list;
         }
+        public HttpResponseMessage GetARMTemplate(string templateName)
+        {
+            var list = TemplatesManager.GetTemplates().ToList();
+            var template = list.Where((temp) => temp.Name == templateName);
+            if (template.Any())
+            {
+                var armTemplateJson = TemplatesManager.GetARMTemplate(template.FirstOrDefault());
```

Comment:
If you just care about FirstOrDefault anyway, then why not just use it instead of Where above then check for null?